private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -316,6 +316,7 @@ func run(
 		nextRevision,
 		outputLanguages,
 		pluginConfig.SPDXLicenseID,
+		pluginConfig.LicenseURL,
```

Comment:
I did not validate `pluginConfig.LicenseURL` because `pluginConfig.SourceURL` is also not validated